_component_design/switches.md

```diff
@@ -14,72 +15,91 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
-<!--{% include todo.html content="Update focus states, expand on inline validation, and improve
-keyboard accessibility." %}-->
 
-The switch should only be used in **mobile** designs. Enabling preferences or
-settings is a prime use case for this component. The switch can be used with or
-without text, depending on the context provided in the switch label. If more
-complex text is needed in the switch than yes/no or on/off, use a **toggle**.
+## Introduction
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-## Default
+### When to use
+{: .hxSubSectionTitle}
```

Comment:
Forgot to note: We're taking these class name calls out per @araiford: https://github.com/rackerlabs/design-system/pull/292#issuecomment-338716258

Reply:
removed across the doc.